sdk/tests/conformance2/extensions/webgl-shader-pixel-local-storage.html

```diff
@@ -67,8 +67,10 @@
   checkInitialValues();
   checkWebGLNonNormativeBehavior();
 
-  checkRendering(gl);
-  checkRendering(gl_no_alpha);
+  await checkRendering(gl);
+  await checkRendering(gl_no_alpha);
+
+  finishTest();
```

Comment:
Just for my own understanding, how does the test harness know that the test is async now and that it should wait for `finishTest`? Before was it assuming the test was over as soon as it finished parsing?

Reply:
`js-test-post.js` notifies the test harness explicitly. When included statically, it's called as soon as the parsing is finished. With `finishTest`, the same script is added on the fly.